pkgs/applications/science/physics/crystfel/add-meson-build.patch

```diff
@@ -0,0 +1,116 @@
+diff --git a/meson.build b/meson.build
+new file mode 100644
+index 0000000..8fc0631
+--- /dev/null
++++ b/meson.build
+@@ -0,0 +1,110 @@
++# Meson file for libccp4c (CCP4 core libraries, C part only)
```

Comment:
If we add the entire file we can skip the patch part which adds some nice highlighting